### ISSUE TITLE
Change background color of tiles when map is loading

### ISSUE THREAD

**Issue by mazing** [tomchentw/react-google-maps#567]
I'm trying to change the color (i.e. the gray color) of the tiles when the map is loading.

In https://developers.google.com/maps/documentation/javascript/reference#MapOptions, it seems it's possible to change the background color of the map. However, I cannot find a way to do this with `react-google-maps`.

**Comment by oshalygin** [comment]
```jsx
    const defaultOptions = {
      styles: [mapStyles],
      mapTypeId: additionalProps.mapTypeId,
      mapTypeControl: false,
      zoomControl: true,
      streetViewControl: true,
      draggableCursor,
      draggingCursor,
    };

    return (
      <GoogleMap
        defaultOptions={defaultOptions}
        options={{  // <-- Use this
          ...defaultOptions,
          draggableCursor: additionalProps.cursor,
        }}
        loadingElement={<div />}
        ref={onMapLoad}
      >
        {markers}
      </GoogleMap>
```

**Comment by Talmaj** [comment]
I'm trying to do this as well, however it doesn't work. @mazing did it work for you?

**Comment by ascii-husky** [comment]
This does not work. 
Maybe because the options are set via setOptions and backgroundColor can only be set initially? 
https://developers.google.com/maps/documentation/javascript/reference/3/map#MapOptions

**Comment by Talmaj** [comment]
I created a pull request #765 that resolved this issue and #764 for me.

**Comment by sanaali-mitkat** [comment]
this is not working
